mp/src/game/server/player_command.cpp

```diff
@@ -426,6 +431,44 @@ void CPlayerMove::RunCommand ( CBasePlayer *player, CUserCmd *ucmd, IMoveHelper
 
     player->m_vecOldOrigin = player->GetAbsOrigin();
 
+	// If the player unducks while in the air, but their feet end up being within sv_considered_on_ground above
+    // standable ground, they can unduck and jump on the same tick to perform a jumpbug. This allows the player to get
+    // grounded and jump, but because triggers are first checked after all player movement is done, it is possible to
+    // exit a trigger the player unducked into with the jump. This has do be done here because ProcessImpacts()
+    // uses the player entity and not the move data that is used in the game movement.
+    if (sv_ground_trigger_fix.GetBool() &&
+        player->GetGroundEntity() == nullptr && !static_cast<CMomentumPlayer*>(player)->m_CurrentSlideTrigger &&                         // In air
+        player->GetFlags() & FL_DUCKING && player->m_afButtonReleased & IN_DUCK &&                                                       // Tries to unduck
+        (player->m_afButtonPressed & IN_JUMP || (static_cast<CMomentumPlayer*>(player)->HasAutoBhop() && player->m_nButtons & IN_JUMP))) // Tries to jump
```

Comment:
I don't like the multiple static_casts in the same expression. Just create a new `pMomPlayer` variable above and use it,